xray/utils/analyzermanager.go

```diff
@@ -96,7 +103,11 @@ type AnalyzerManager struct {
 	AnalyzerManagerFullPath string
 }
 
-func (am *AnalyzerManager) Exec(configFile, scanCommand string, serverDetails *config.ServerDetails) (err error) {
+func (am *AnalyzerManager) GetAnalyzerManagerDir() string {
```

Comment:
i think it's redundant to have a func that runs a one line func, WDYT?

Reply:
as explained above, we have to do it because it depends on what scan you run.

the analyzerManager runs the binary.
the scanner provide the arguments using the func, different for each scan.
the commands, the working dir are different...